cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -38,6 +40,17 @@ module Verifier.SAW.Cryptol
   , defaultPrimitiveOptions
   , genNominalConstructors
   , exportValueWithSchema
+
+  -- FIXME: the following are unused in the project; currently we export
+  -- to turn off warnings, but ...
+  --   Q. Are these detritus or,
+  --      are these here for future clients of this module?
+  , isCryptolModuleName
+  , isCryptolInteractiveName
+  , pIsNeq
+  , fvAsBool
+  , exportFirstOrderValue
+  , importFirstOrderValue
```

Comment:
I have noticed this as well, but I lean more on the side of these being leftover detritus that someone forgot to remove after refactoring the code in `cryptol-saw-core` in a way that rendered these functions unused. For example, I could envision the `pIsNeq` function being used at some point, but for whatever reason, it is no longer used today.

I'd favor just removing these functions. We can always restore them later if need be.

Reply:
removal on it's way.